src/main/webapp/homepage_style.css

```diff
@@ -65,6 +69,17 @@ section {
   padding-bottom: 1.5em;
 }
 
+#control-bar-message-wrapper {
+  width: 100%;
+  max-width: 55em;
+  z-index: 11;
```

Comment:
It seems that this is the only z-index element. I may be mistaken, but I believe setting the value to 1 should have the same effect as a lone z-index with a value of 11

Reply:
Ah yes, I need to stop using random numbers for this, I'll change this to 1 since it's the only one there. Thanks!